data/json/npcs/TALK_ALLY_TUTORIAL.json

```diff
@@ -127,8 +137,11 @@
   {
     "id": "TALK_ALLY_TUTORIAL_GUARD",
     "type": "talk_topic",
-    "dynamic_line": "You tell me to guard, I'll stay where I am and guard it - unless I'm in a vehicle, in which case I'll stick with the vehicle.\n  I'm not a potted plant, though, so if I hear something dangerous happening, I'm going to go see what it is instead of getting jumped in the dark.  If you want me to stay put, tell me not to investigate noises - but if I get shot by some bandit because I can't see where he is and you don't want me to go looking, I won't be happy.\n  You can also use the zone manager (keybind 'Y') to set up no-investigate zone, so if there's some monsters behind a door that you know about, I can ignore them.  You can also set on an investigate-only zone, and I won't investigate noises coming from outside the zone.  The no-investigate zone takes precedence over the investigate-only, if there's a noise coming from some place in both zones.  And if you've got an investigate-only zone set anywhere, even if it's far away, I won't investigate noises coming from outside of it, so be careful with those zones.  Like I said, I don't want to get sniped by some bandit because you told me not to go looking for him - but I also don't want to go investigate something pounding at door only to find out it's some horrible monster you forgot to warn me about.",
-    "responses": [ { "text": "Just in case - how else can I tell you to stay put?", "topic": "TALK_ALLY_TUTORIAL_MISC" } ]
+    "dynamic_line": "You tell me to guard, I'll stay where I am and guard it - unless I'm in a vehicle, in which case I'll stick with the vehicle.  I might not be able to sop a horde of zombies, but I can keep people other people from stealing our stuff.  Well, unless they try to kill me.\n  Also, I'm not a potted plant, so if I hear something dangerous happening, I'm going to go see what it is instead of getting jumped in the dark.  If you want me to stay put, tell me not to investigate noises - but if I get shot by some bandit because I can't see where he is and you don't want me to go looking, I won't be happy.\n  You can also use the zone manager (keybind 'Y') to set up no-investigate zone, so if there's some monsters behind a door that you know about, I can ignore them.  You can also set on an investigate-only zone, and I won't investigate noises coming from outside the zone.  The no-investigate zone takes precedence over the investigate-only, if there's a noise coming from some place in both zones.  And if you've got an investigate-only zone set anywhere, even if it's far away, I won't investigate noises coming from outside of it, so be careful with those zones.  Like I said, I don't want to get sniped by some bandit because you told me not to go looking for him - but I also don't want to go investigate something pounding at door only to find out it's some horrible monster you forgot to warn me about.",
```

Comment:
```suggestion
    "dynamic_line": "You tell me to guard, I'll stay where I am and guard it - unless I'm in a vehicle, in which case I'll stick with the vehicle.  I might not be able to stop a horde of zombies, but I can keep people other people from stealing our stuff.  Well, unless they try to kill me.\n  Also, I'm not a potted plant, so if I hear something dangerous happening, I'm going to go see what it is instead of getting jumped in the dark.  If you want me to stay put, tell me not to investigate noises - but if I get shot by some bandit because I can't see where he is and you don't want me to go looking, I won't be happy.\n  You can also use the zone manager (keybind 'Y') to set up no-investigate zone, so if there's some monsters behind a door that you know about, I can ignore them.  You can also set on an investigate-only zone, and I won't investigate noises coming from outside the zone.  The no-investigate zone takes precedence over the investigate-only, if there's a noise coming from some place in both zones.  And if you've got an investigate-only zone set anywhere, even if it's far away, I won't investigate noises coming from outside of it, so be careful with those zones.  Like I said, I don't want to get sniped by some bandit because you told me not to go looking for him - but I also don't want to go investigate something pounding at door only to find out it's some horrible monster you forgot to warn me about.",
```